src/dsl/orbit-codegen/src/main/kotlin/cloud/orbit/dsl/java/TypeIndexer.kt

```diff
@@ -20,14 +16,17 @@ internal class TypeIndexer : AstVisitor() {
         Type("float") to TypeName.FLOAT,
         Type("int32") to TypeName.INT,
         Type("int64") to TypeName.LONG,
-        Type("string") to ClassName.get(String::class.java)
+        Type("void") to TypeName.VOID,
```

Comment:
Is `void` new? Tests for it? What happens when you use it in a parameter?

Reply:
It's new, but I hadn't considered the parameter case. I'll remove it and add support for it in a separate change covering all the things.